src/Umbraco.Web.UI.Client/src/common/directives/components/tree/umbtreeitem.directive.js

```diff
@@ -65,13 +65,19 @@ angular.module("umbraco.directives")
                 }
             }
 
+            function isRtl()
+            {
+                var x = document.dir;

```

Comment:
I guess the var x was used for debugging? Seems superfluous to add a new var, could just `return document.dir === "rtl" ? true : false;`, right?